django_stubs_ext/dev-requirements.txt

```diff
@@ -0,0 +1,7 @@
+wheel
+mypy==0.790
+pre-commit==2.7.1
+pytest==6.1.1
+-e .
+
+#gitpython==3.1.9
```

Comment:
Can we remove this line?

Reply:
🤔 

Do we even need this file?

Reply:
you're right. monorepo, all the development is gonna be done from within the main repo :p